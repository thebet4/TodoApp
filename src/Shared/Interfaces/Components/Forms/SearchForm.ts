import { StyleProp, ViewStyle } from 'react-native';

export default interface ISearchFormProps {
  style?: StyleProp<ViewStyle>;
}
