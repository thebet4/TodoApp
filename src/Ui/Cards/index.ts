export { default as HotelCard } from './HotelCard';
export { default as TeamCard } from './TeamCard';
