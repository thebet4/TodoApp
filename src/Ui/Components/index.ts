export { default as Header } from './Header'
export { default as TabBarItem } from './TabBarItem'
