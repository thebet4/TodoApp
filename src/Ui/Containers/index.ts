export { default as Container } from './Container';
export { ContentContainer } from './ContentContainer';
