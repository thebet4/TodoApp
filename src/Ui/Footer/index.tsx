import React from 'react';
import {
  ImageBackground, View, TouchableOpacity, Image,
} from 'react-native';
import { Colors } from '../Theme';
import Caption from '../Typography/Caption';
import Title40 from '../Typography/Title40';

const Footer = () => {
  return (

    <ImageBackground
      source={require('./Assets/HomeFooterBg.png')}
      style={{ paddingBottom: 64 }}
    >
      <View style={{
        flexDirection: 'row',
        alignItems: 'center',
        justifyContent: 'center',
        marginTop: 92,
      }}
      >
        <View style={{
          width: '30%',
          backgroundColor: Colors.Base.Light,
          height: 1,
        }}
        />
        <Title40 style={{
          color: Colors.Base.Light,
          marginHorizontal: 12,
        }}
        >
          Partners
        </Title40>
        <View style={{
          width: '30%',
          backgroundColor: Colors.Base.Light,
          height: 1,
        }}
        />
      </View>

      <View style={{
        alignItems: 'center',
        justifyContent: 'space-around',
      }}
      >
        <Image source={require('./Assets/partner1.png')} resizeMode="center" />
        <Image source={require('./Assets/partner2.png')} resizeMode="center" />
        <Image source={require('./Assets/partner4.png')} resizeMode="center" />
        <Image source={require('./Assets/partner5.png')} resizeMode="center" />
        <Image source={require('./Assets/partner6.png')} resizeMode="center" />
        <Image source={require('./Assets/partner7.png')} resizeMode="center" />

      </View>

      <View style={{ width: '70%', alignSelf: 'center' }}>

        <TouchableOpacity style={{
          height: 44,
          alignItems: 'center',
          justifyContent: 'center',
          borderWidth: 2,
          borderColor: Colors.Base.Light,
          borderRadius: 51,
          marginBottom: 17,
        }}
        >
          <Caption style={{ color: Colors.Base.Light, fontSize: 18 }}>
            Acessar
          </Caption>
        </TouchableOpacity>
        <TouchableOpacity style={{
          height: 44,
          alignItems: 'center',
          justifyContent: 'center',
          borderRadius: 51,
          backgroundColor: Colors.Base.Light,
        }}
        >
          <Caption style={{ color: Colors.Primary.Normal, fontSize: 18 }}>
            Abrir Conta
          </Caption>
        </TouchableOpacity>
      </View>
    </ImageBackground>

  );
};

export default Footer;
