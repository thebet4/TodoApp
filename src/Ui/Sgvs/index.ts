export { default as CalendarIcon } from './CalendarIcon'
export { default as ListIcon } from './ListIcon'
export { default as SunIcon } from './SunIcon'
export { default as TaskIcon } from './TaskIcon'
export { default as SearchIcon } from './SearchIcon'
export { default as FilterIcon } from './FilterIcon'
export { default as AddIcon } from './AddIcon'
export { default as PersonalIcon } from './PersonalIcon'
export { default as PriorityIcon } from './PriorityIcon'
export { default as ClockIcon } from './ClockIcon'
export { default as CommentIcon } from './CommentIcon'
export { default as RepeatIcon } from './RepeatIcon'
