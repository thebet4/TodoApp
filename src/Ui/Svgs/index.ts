export { default as LogoIcon } from './LogoIcon';
export { default as OpenedEyeIcon } from './OpenedEyeIcon';
export { default as CalendarIcon } from './CalendarIcon';
export { default as SearchIcon } from './SearchIcon';
export { default as NFTIcon } from './NFTIcon';
export { default as HotelIcon } from './HotelIcon';
export { default as BlockChainIcon } from './BlockChainIcon';
export { default as LogoWithTextIcon } from './LogoWithTextIcon';
