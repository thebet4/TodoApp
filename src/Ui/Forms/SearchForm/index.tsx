import React from 'react';
import { View } from 'react-native';
import { CalendarIcon, OpenedEyeIcon } from '../../Svgs';
import { Colors } from '../../Theme';
import Caption from '../../Typography/Caption';
import {
  FormContainer, HorizontalDivider, VerticalContainer, VerticalDivider,
} from './style';

const SearchForm = () => {
  return (
    <FormContainer>
      <VerticalContainer>
        <View style={{
          flexDirection: 'row',
          alignItems: 'center',
        }}
        >
          <OpenedEyeIcon
            color={Colors.Primary.Normal}
            width={32}
            height={32}
            style={{ marginRight: 12 }}
          />
          <Caption style={{ color: Colors.Base.Gray }}>
            Para onde quer ir?
          </Caption>
        </View>
      </VerticalContainer>
      <HorizontalDivider />
      <VerticalContainer>
        <View style={{
          flexDirection: 'row',
          alignItems: 'center',
        }}
        >
          <CalendarIcon
            color={Colors.Primary.Normal}
            width={24}
            height={24}
            style={{ marginRight: 12 }}
          />
          <Caption style={{ color: Colors.Base.Gray }}>Entrada</Caption>
        </View>

        <VerticalDivider />
        <View style={{
          flexDirection: 'row',
          alignItems: 'center',
        }}
        >
          <CalendarIcon
            color={Colors.Primary.Normal}
            width={24}
            height={24}
            style={{ marginRight: 12 }}
          />
          <Caption style={{ color: Colors.Base.Gray }}>Saida</Caption>
        </View>
      </VerticalContainer>
    </FormContainer>
  );
};

export default SearchForm;
