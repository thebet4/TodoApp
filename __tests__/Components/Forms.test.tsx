import renderer from 'react-test-renderer';
import React from 'react';

import { SearchForm } from '../../src/Ui/Forms';

describe('<SearchForm />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SearchForm />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
