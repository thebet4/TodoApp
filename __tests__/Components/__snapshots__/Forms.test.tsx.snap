// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchForm /> renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderRadius": 32,
        "justifyContent": "center",
        "minHeight": 100,
        "width": "100%",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 64,
          "justifyContent": "space-evenly",
          "width": "100%",
        },
      ]
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight={32}
        bbWidth={32}
        fill="none"
        focusable={false}
        height={32}
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            Object {
              "marginRight": 12,
            },
            Object {
              "flex": 0,
              "height": 32,
              "width": 32,
            },
          ]
        }
        vbHeight={32}
        vbWidth={32}
        width={32}
      >
        <RNSVGGroup
          fill={null}
          propList={
            Array [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M1.33334 16C1.33334 16 6.66668 5.33333 16 5.33333C25.3333 5.33333 30.6667 16 30.6667 16C30.6667 16 25.3333 26.6667 16 26.6667C6.66668 26.6667 1.33334 16 1.33334 16Z"
            propList={
              Array [
                "stroke",
              ]
            }
            stroke={4285140475}
          />
          <RNSVGPath
            d="M16 20C18.2091 20 20 18.2091 20 16C20 13.7909 18.2091 12 16 12C13.7909 12 12 13.7909 12 16C12 18.2091 13.7909 20 16 20Z"
            propList={
              Array [
                "stroke",
              ]
            }
            stroke={4285140475}
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Inter_700Bold",
              "fontSize": 16,
              "lineHeight": 17.92,
            },
            Object {
              "color": "#585858",
            },
          ]
        }
      >
        Para onde quer ir?
      </Text>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#6A0DFB",
          "height": 1,
          "width": "100%",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 64,
          "justifyContent": "space-evenly",
          "width": "100%",
        },
      ]
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
        }
      }
    >
      <RNSVGSvgView
        bbHeight={24}
        bbWidth={24}
        fill="none"
        focusable={false}
        height={24}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            Object {
              "marginRight": 12,
            },
            Object {
              "flex": 0,
              "height": 24,
              "width": 24,
            },
          ]
        }
        width={24}
      >
        <RNSVGGroup
          fill={null}
          propList={
            Array [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M19 4H5C3.89543 4 3 4.89543 3 6V20C3 21.1046 3.89543 22 5 22H19C20.1046 22 21 21.1046 21 20V6C21 4.89543 20.1046 4 19 4Z"
            propList={
              Array [
                "stroke",
              ]
            }
            stroke={4285140475}
          />
          <RNSVGPath
            d="M16 2V6"
            propList={
              Array [
                "stroke",
              ]
            }
            stroke={4285140475}
          />
          <RNSVGPath
            d="M8 2V6"
            propList={
              Array [
                "stroke",
              ]
            }
            stroke={4285140475}
          />
          <RNSVGPath
            d="M3 10H21"
            propList={
              Array [
                "stroke",
              ]
            }
            stroke={4285140475}
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Inter_700Bold",
              "fontSize": 16,
              "lineHeight": 17.92,
            },
            Object {
              "color": "#585858",
            },
          ]
        }
      >
        Entrada
      </Text>
    </View>
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#6A0DFB",
            "height": 64,
            "width": 1,
          },
        ]
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
        }
      }
    >
      <RNSVGSvgView
        bbHeight={24}
        bbWidth={24}
        fill="none"
        focusable={false}
        height={24}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            Object {
              "marginRight": 12,
            },
            Object {
              "flex": 0,
              "height": 24,
              "width": 24,
            },
          ]
        }
        width={24}
      >
        <RNSVGGroup
          fill={null}
          propList={
            Array [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M19 4H5C3.89543 4 3 4.89543 3 6V20C3 21.1046 3.89543 22 5 22H19C20.1046 22 21 21.1046 21 20V6C21 4.89543 20.1046 4 19 4Z"
            propList={
              Array [
                "stroke",
              ]
            }
            stroke={4285140475}
          />
          <RNSVGPath
            d="M16 2V6"
            propList={
              Array [
                "stroke",
              ]
            }
            stroke={4285140475}
          />
          <RNSVGPath
            d="M8 2V6"
            propList={
              Array [
                "stroke",
              ]
            }
            stroke={4285140475}
          />
          <RNSVGPath
            d="M3 10H21"
            propList={
              Array [
                "stroke",
              ]
            }
            stroke={4285140475}
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Inter_700Bold",
              "fontSize": 16,
              "lineHeight": 17.92,
            },
            Object {
              "color": "#585858",
            },
          ]
        }
      >
        Saida
      </Text>
    </View>
  </View>
</View>
`;
